src/index.html

```diff
@@ -1,13 +1,425 @@
+              grid__item--tablet-1-3
+              grid__item--desktop-1-6"
+          >
+            Timeless, for 50 years and counting
+          </h2>
+          <p
+            class="timeless__text
+            grid__item
+            grid__item--mobile-1-2
+            grid__item--tablet-4-6
+            grid__item--desktop-8-12"
+          >
+            Bang & Olufsen is expanding its Recreated Classics Program with the launch of Beosystems – a limited edition music system that transcends time by bridging the gap between one of our iconic designs from 1972 and today’s cutting-edge digital technology.
+          </p>
+        </div>
+
+      </section>
+
+      <section id="contact-us" class="page__section contact-us">
+        <div class="container">
+          <h2 class="page__section-title">Contact us</h2>
+          <div class="grid">
+            <form
+              onsubmit="this.reset(); return false;"
+              action="/"
+              method="get"
+              class="
+                grid__item
+                grid__item--mobile-1-2
+                grid__item--tablet-1-6
+                grid__item--desktop-1-5
+                contact-us__form"
+            >
+              <input
+                name="name"
+                title="Введіть своє ім'я"
+                type="text"
+                placeholder="Name"
+                class="contact-us__input"
+                required
+              >
+              <input
+                name="email"
+                title="Введіть адресу електронної пошти"
+                type="email"
+                placeholder="Email"
+                class="contact-us__input"
+                required
+              >
+              <textarea
+                id="textarea"
+                title="Введіть своє повідомлення"
+                name="message"
+                placeholder="Message"
+                class="contact-us__input contact-us__input--textarea"
+                required
+              ></textarea>
+              <input
+                type="submit"
+                class="button"
+                value="Send"
+              >
+            </form>
+
+            <div
+              class="
+                grid__item
+                grid__item--mobile-1-2
+                grid__item--tablet-1-6
+                grid__item--desktop-8-12
+                contact-us__contacts"
+            >
+              <div class="contact-us__item">
+                <div class="contact-us__title">
+                  Phone
+                </div>
+                <a
+                  href="tel: +1 234 5555-55-55"
+                  class="contact-us__link"
+                >
+                  +1 234 5555-55-55
+                </a>
+              </div>
+
+              <div class="contact-us__item">
+                <div class="contact-us__title">
+                  Email
+                </div>
+                <a
+                  href="mailto: hello@bang&olufsen.com"
```

Comment:
The email address should be URL-encoded. Replace the '&' symbol with '%26' in the mailto link